src/libs/sso-utils.js

```diff
@@ -37,6 +37,26 @@ export const checkCredentialValid = credentials => {
 export const checkUserProfile = userInfo =>
   !(!userInfo.email || !userInfo.firstName || !userInfo.lastName);
 
+export const checkArray = array => !Array.isArray(array) || !array.length;
+
+export const checkRocketChatSchema = async userInfo => {
```

Comment:
does this func need to be async? doesn't seem to have anything async about it

Reply:
yes I forgot to await for `isUserInOrgs`